_test/test_dnd_class/test_dnd_metadata_data.m

```diff
@@ -67,14 +67,16 @@ function test_d2d_metadata_get_set(~)
             assertTrue(isa(dnd_obj,'d2d'));
 
             dnd_met = dnd_obj.metadata;
+            dtt = datetime("now");
+
 
             input{1}.label = dnd_obj.label;
             assertEqual(dnd_met.axes,input{1});
             assertEqual(dnd_met.proj,input{2});
-            % save operation saves metadata with current date
+            % save or get_metadata operation generates metadata with current
+            % date  if it has not been defined earlier
             assertTrue(dnd_met.creation_date_defined);
 
-            dtt = datetime("now");
             assertEqual(dnd_met.creation_date_str, ...
```

Comment:
Would it be better to make an almost equal for these dates? Can we just extract day/month/year hour/minute? Or can we just do a `timeA - timeB < 5sec` operation?

Reply:
The functions return strings and results compared as strings. May be would be reasonable to define comparison opeator for datetime (or our custom `equal_to_tol` implementation) to have tolerance in second, minutes etc, but this work is far beyond the scope of this ticket.